### ISSUE TITLE
Can't deminimize the window that minimized manually

### ISSUE THREAD

**Issue by ErwinLiYH** [koekeishiya/yabai#2153]
Version: yabai-v6.0.15

I manually minimize a window by the left-top button of the window, then this window can't be deminimized by command `yabai -m window --deminimize <WIN_ID>`. I must focus on another window in this space, and then deminimize the window by command.

To reproduce:

1. record a test window's ID, here is 32361
2. make some shortcuts for the test
    ```
    shift + cmd -z : yabai -m query --windows --window &> /Users/yuanhaoli/Desktop/yabai-window.txt
    shift + cmd -x : yabai -m query --windows --space &> /Users/yuanhaoli/Desktop/yabai-space.txt
    shift + cmd -c : yabai -m window --deminimize 32361 &> /Users/yuanhaoli/Desktop/yabai-demini.txt
    ```
3. minimize the window by the left-top button
4. run these three shortcuts to get debug information and try to deminimize the window

Result:

```
// yabai-window.txt
could not retrieve window details.
```

```
// yabai-space.txt
[{
	"id":32361,
	"pid":19499,
	"app":"Code",
	"title":"yabai-space.txt — yabai_manager",
	"frame":{
		"x":6.0000,
		"y":44.0000,
		"w":1500.0000,
		"h":932.0000
	},
	"role":"AXWindow",
	"subrole":"AXStandardWindow",
	"root-window":true,
	"display":1,
	"space":2,
	"level":0,
	"sub-level":0,
	"layer":"normal",
	"sub-layer":"normal",
	"opacity":1.0000,
	"split-type":"none",
	"split-child":"none",
	"stack-index":0,
	"can-move":true,
	"can-resize":true,
	"has-focus":true,
	"has-shadow":true,
	"has-parent-zoom":false,
	"has-fullscreen-zoom":false,
	"is-native-fullscreen":false,
	"is-visible":false,
	"is-minimized":true,
	"is-hidden":false,
	"is-floating":false,
	"is-sticky":false,
	"is-grabbed":false
}]
```

```
// yabai-demini.txt
could not locate the window to act on!
```

yabai-window.txt shows the command result of `yabai -m query --windows --window`, which may be means no focused window in the space, but at the same time, the result in yabai-space.txt shows the window is focused (`"has-focus":true`) and minimized (`"is-minimized":true`). yabai-demini.txt shows the result of `yabai -m window --deminimize 32361`, which is `could not locate the window to act on!`.

**Comment by koekeishiya** [comment]
Thanks; fixed on master.

**Comment by koekeishiya** [comment]
Fixed in v7.0.0.
